aresponses/main.py

```diff
@@ -103,6 +111,32 @@ def _host_matches(self, match_host):
         self._loop.stop()
         raise self._exception  # noqa
 
+    async def make_network_request(self, request):
+        """Make non-mocked network request"""
+        connector = TCPConnector()
+        connector._resolve_host = partial(self._old_resolver_mock, connector)
+
+        new_is_ssl = ClientRequest.is_ssl
+        ClientRequest.is_ssl = self._old_is_ssl
+        try:
+            original_request = request.clone(
+                scheme='https' if request.headers['AResponsesIsSSL'] else 'http')
+
+            headers = {k: v for k, v in request.headers.items() if k != 'AResponsesIsSSL'}
+
+            async with ClientSession(connector=connector) as session:
+                async with getattr(session, request.method.lower())(
+                        original_request.url, headers=headers,
+                        data=(await request.read())
+                ) as r:
+                    headers = {k: v for k, v in r.headers.items() if k.lower() == 'content-type'}
```

Comment:
Why is content-type treated differently?

Reply:
In future, there should be some blacklist of headers, which are handled internally by the proxy's HTTP client (e.g. 'content-encoding') and therefore they should not be forwarded to the tested client. For example, if the 'content-encoding' is 'gzip', the proxy's client decodes the data, so the response for the tested client is already decoded, and the 'content-encoding' response header should not be passed through. There are more such headers, which I did not really understand how to handle (I'm not very pro in HTTP), so I've just whitelisted the 'Content-Type', which is sufficient in many cases (e.g. to be able to pass-through JSONs).